src/PHPUnit/Controller/AbstractControllerTestCase.php

```diff
@@ -194,30 +183,24 @@ public function getApplication()
 
     /**
      * Get the service manager of the application object
-     *
-     * @return ServiceManager
      */
-    public function getApplicationServiceLocator()
+    public function getApplicationServiceLocator(): ServiceManager
```

Comment:
Can this be typehinted with PSR container instead?

Reply:
done

Reply:
It should the `Psr\Container\ContainerInterface` and not any interface from Laminas or laminas-servicemanager.

Reply:
done

Reply:
@froschdesign : this change make the build fail, can you check this ?